src/librustc_lint/lib.rs

```diff
@@ -278,11 +278,19 @@ pub fn register_builtins(store: &mut lint::LintStore, sess: Option<&Session>) {
             reference: "issue #46906 <https://github.com/rust-lang/rust/issues/46906>",
             epoch: None,
         },
-         FutureIncompatibleInfo {
-             id: LintId::of(lint::builtin::BARE_TRAIT_OBJECT),
-             reference: "issue #48457 <https://github.com/rust-lang/rust/issues/48457>",
-             epoch: Some(session::config::Epoch::Epoch2018),
-         }
+        FutureIncompatibleInfo {
+            id: LintId::of(lint::builtin::BARE_TRAIT_OBJECT),
+            reference: "issue #48457 <https://github.com/rust-lang/rust/issues/48457>",
+            epoch: Some(session::config::Epoch::Epoch2018),
+        },
+        FutureIncompatibleInfo {
```

Comment:
I agree that some of the text for `FutureIncompatibleInfo` here is suboptimal. We could easily special case this particular lint-id, though... which ...seems fine.